src/getHumanAge.test.js

```diff
@@ -7,4 +7,65 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test ('the result should return an array', () => {
+    const result = getHumanAge();
+    expect(result)
+    .toBeInstanceOf(Array);
+  });
+
+  test ('should return an array with two elements', () => {
+    const result = getHumanAge();
+    expect(result.length)
+    .toBe(2);
+  });
+
+ 
+  test ('should return zeros if both variables are zeros', () => {
+    const result = getHumanAge(0,0);
+    expect(result)
+    .toEqual([0,0]);
+  });
+
+  test ('should return zeros when animals ages are less then 15', () => {
+    const result = getHumanAge(14,14);
+    expect (result)
+    .toEqual([0,0]);
+  });
+  test ('should return one year of human when animals ages are equal 15', () => {
+    const result = getHumanAge(15,15);
+    expect (result)
+    .toEqual([1,1]);
+  });
+
+  test ('should return one year of human when animals ages are less then 24', () => {
```

Comment:
The test case description is misleading. It should say 'should return [1,1] when animals ages are less than 24'. Please update the test case description to match the expected output.